### PR TITLE
Add a cymbal-toy-store demo to the devrel-demos

### PR DESCRIPTION
**Pull request by gotochkin**
Add the cymbal-toy-store demo to the devrel-demos repository

### CODE REVIEW

File: infrastructure/cymbal-toy-store/frontend/public/hero.png
Comment (dtest) [inline infrastructure/cymbal-toy-store/frontend/public/hero.png:1]: We should stay away from baby/kid pictures, right?

File: infrastructure/cymbal-toy-store/frontend/src/app/chat.tsx
Comment (dtest) [inline infrastructure/cymbal-toy-store/frontend/src/app/chat.tsx:1]: This file should have license at the top.

File: infrastructure/cymbal-toy-store/frontend/src/app/api.tsx
Comment (dtest) [inline infrastructure/cymbal-toy-store/frontend/src/app/api.tsx:1]: This file should have license at the top.

File: infrastructure/cymbal-toy-store/frontend/src/app/layout.tsx
Comment (dtest) [inline infrastructure/cymbal-toy-store/frontend/src/app/layout.tsx:1]: This file should have license at the top.

File: infrastructure/cymbal-toy-store/frontend/src/app/page.tsx
Comment (dtest) [inline infrastructure/cymbal-toy-store/frontend/src/app/page.tsx:1]: This file should have license at the top.

File: infrastructure/cymbal-toy-store/README.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+* Get endpoint from output of the previous command, from the web console for cloud run or using the following command:
+```
+gcloud  run services list --filter="(cymbal-toystore)" --format="value(URL)"
+```
+* Access to the endpoint
+- You can open the endpoint to the public access 
+- You can use proxy 
+- You can put a load balancer before it
+
+# Licence
```
Comment (dtest) [inline infrastructure/cymbal-toy-store/README.md]: nit: License

File: infrastructure/cymbal-toy-store/README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+- The application is using native integration between Google Databases and LangChain 
+
+### Architecture
+
+![Overview](./cymbal-toystore-01.png)
+
+## Requirements
+- Project in Google Cloud with enabled APIs for all components.
+- AlloyDB cluster and primary instance created 
+- A laptop or a VM with Python 3.11+,git and postggres utilities such as psql
```
Comment (dtest) [inline infrastructure/cymbal-toy-store/README.md]: nit: postgres (not postggres)

File: infrastructure/cymbal-toy-store/Dockerfile
Comment (dtest) [inline infrastructure/cymbal-toy-store/Dockerfile:1]: This file should have license at the top.

File: infrastructure/cymbal-toy-store/frontend/public/hero.png
Comment (gotochkin) [reply infrastructure/cymbal-toy-store/frontend/public/hero.png:1]: Deleted

File: infrastructure/cymbal-toy-store/frontend/src/app/chat.tsx
Comment (gotochkin) [reply infrastructure/cymbal-toy-store/frontend/src/app/chat.tsx:1]: Done

File: infrastructure/cymbal-toy-store/frontend/src/app/api.tsx
Comment (gotochkin) [reply infrastructure/cymbal-toy-store/frontend/src/app/api.tsx:1]: Done

File: infrastructure/cymbal-toy-store/frontend/src/app/layout.tsx
Comment (gotochkin) [reply infrastructure/cymbal-toy-store/frontend/src/app/layout.tsx:1]: Done

File: infrastructure/cymbal-toy-store/frontend/src/app/page.tsx
Comment (gotochkin) [reply infrastructure/cymbal-toy-store/frontend/src/app/page.tsx:1]: Done

File: infrastructure/cymbal-toy-store/README.md (hunk 1 above)
Comment (gotochkin) [reply infrastructure/cymbal-toy-store/README.md]: Fixed.

File: infrastructure/cymbal-toy-store/README.md (hunk 2 above)
Comment (gotochkin) [reply infrastructure/cymbal-toy-store/README.md]: Fixed.

File: infrastructure/cymbal-toy-store/Dockerfile
Comment (gotochkin) [reply infrastructure/cymbal-toy-store/Dockerfile:1]: Done.

**Review by dtest — APPROVED** [review]
LGTM

**Pull request merged**